data_steward/tools/create_combined_dataset.py

```diff
@@ -0,0 +1,234 @@
+"""
```

Comment:
Very similar to `create_tier.py`.  For future work, we may see how to combine the two scripts more fully.